src/pybind/mgr/dashboard/controllers/pool.py

```diff
@@ -74,6 +74,7 @@
     "fast_read": (bool, ""),
     "options": ({
         "pg_num_min": (int, "")
+        "pg_num_max": (int, "")
```

Comment:
I think this should address https://jenkins.ceph.com/job/ceph-pull-requests/86047/consoleFull#450684626c19247c4-fcb7-4c61-9a5d-7e2b9731c678

```suggestion
        "pg_num_min": (int, ""),
        "pg_num_max": (int, "")
```